lib/internal/bootstrap/node.js

```diff
@@ -236,6 +236,11 @@ if (!config.noBrowserGlobals) {
 
   defineOperation(global, 'queueMicrotask', queueMicrotask);
 
+  defineLazyGlobal(global, 'performance', () => {
```

Comment:
do we really need to make it lazy?

Reply:
It's... complicated. Making it global without it being lazy requires a more involved change at the c++ level to register the external references or building the node.js snapshot during build will fail with an illegal instruction. If there's agreement to making this global, we can go back later to make that larger change, which can be done as a semver-patch.

Reply:
I see. Let's keep this for a later change.